config/routes.rb

```diff
@@ -78,7 +81,11 @@
     end
   end
   resources :services, only: [:destroy]
-  resources :service_types, only: [:index]
+  resources :service_types, only: [:index] do
+    collection do
+      get :is_service_working
```

Comment:
If you like, you can change this to `get :is_service_working => 'ui#ui'` and then remove the controller method for this that isn't doing anything meaningful.

Reply:
👍